controllers/azureidentity_controller.go

```diff
@@ -151,8 +151,6 @@ func (r *AzureIdentityReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 	var bindingsToDelete []aadpodv1.AzureIdentityBinding
 	for _, b := range bindings.Items {
-		log = log.WithValues("azureidentitybinding", b.Name)
```

Comment:
This log was never getting used but the side effects from this `WithValues` would have persisted after this loop which doesn't seem right.